src/components/todo/SingleTodo.tsx

```diff
@@ -1,7 +1,17 @@
-import React from "react";
+import { TodoModal } from "../../models";
 
-const SingleTodo = () => {
-  return <div>SingleTodo</div>;
+interface Props {
+  todoSingleObj: TodoModal;
+}
+
+const SingleTodo = ({ todoSingleObj }: Props) => {
```

Comment:
What do you think naming it ``TodoTaskItem``

Reply:
Done